backend/server.js

```
@@ -13,6 +13,14 @@ const {
   handleGracefulShutdown,
   logger
 } = require('./middleware/errorHandler');
+app.use('/api/availability', require('./routes/availabilityRoutes'));
+const { 
+  globalErrorHandler, 
```

Biome (2.1.2)

[error] Shouldn't redeclare 'globalErrorHandler'. Consider to delete it or rename it.

'globalErrorHandler' is defined here:


(lint/suspicious/noRedeclare)